### PR TITLE
Update developer README.md issue #1648

### PR DESCRIPTION
**Pull request by scoomthesnooper**
Added: Note that only the personal-anonymous user can by default have jobs created by the anonymous user.

### CODE REVIEW

**Comment by kannon92** [comment]
Hello @scoomthesnooper!  Welcome to Armada.  Thank you for your first contribution.  

I assigned @Sharpz7 to take a look.

File: docs/dev/README.md [hunk 1]
```diff
@@ -13,4 +13,6 @@ This is not recommended if developing for only Armada.
 We suggest using this if you want to integrate armada with an external service like airflow.
 
 1) Run `docs/dev/setup_local_host.sh`
-2) Run each of those commands in separate terminals
\ No newline at end of file
+2) Run each of those commands in separate terminals
+
+Note: Currently due to End-to-End test rules on the current localdev setup, only the personal-anonymous user can by default have jobs created by anonymous user when using no authentication.
```
Comment (suprjinx) [inline docs/dev/README.md:18]: This document isn't really describing the localdev setup -- so maybe this fact is not needed. I'd say move the change to docs/developer.md

**Review by suprjinx — CHANGES_REQUESTED** [review]

**Comment by scoomthesnooper** [comment]
I have changed it to developer.md i placed it at the bottom of the document as that is just after where it talks about authentication

File: docs/dev/README.md [hunk 2]
```diff
@@ -13,4 +13,5 @@ This is not recommended if developing for only Armada.
 We suggest using this if you want to integrate armada with an external service like airflow.
 
 1) Run `docs/dev/setup_local_host.sh`
-2) Run each of those commands in separate terminals
\ No newline at end of file
+2) Run each of those commands in separate terminals
+
```
Comment (kannon92) [inline docs/dev/README.md:17]: nit: please remove the whitespace.

**Review by Sharpz7 — CHANGES_REQUESTED** [review]
Left a message about slightly re-wording what is written.

Also, if you can add "Closes #issue_id" to your PR message, then the issue will automatically close when the PR gets merged. See more here: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Pull request closed without merging**